File: samples/CS/04-TabGroupHelp/Form1.cs
using System;
using System.Windows.Forms;
using WinForms.Ribbon;

namespace _04_TabGroupHelp
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init(this);

        }
    }
}

File: samples/VB/04-TabGroupHelp/Form1.vb
Imports WinForms.Ribbon

Namespace _04_TabGroupHelp
	Partial Public Class Form1
		Inherits Form
		Private _ribbonItems As RibbonItems

		Public Sub New()
			InitializeComponent()
			_ribbonItems = New RibbonItems(_ribbon)
			_ribbonItems.Init(Me)
		End Sub
	End Class
End Namespace
